pkg/api/helper/helpers.go

```diff
@@ -112,6 +112,18 @@ func IsStandardContainerResourceName(str string) bool {
 	return standardContainerResources.Has(str)
 }
 
+var nonstandardContainerResources = sets.NewString(
```

Comment:
This is called non-standard for what reason?  Is api.ResourceStorageOverlay a standard resource or not?